inc/saiarsprofile.h

```diff
@@ -176,155 +176,219 @@ typedef enum _sai_ars_profile_attr_t
 
     /**
      * @brief Number of quantization bands support for quality map.
+     * Count will be 2 if SAI_ARS_PROFILE_ATTR_PORT_LOAD_PAST and SAI_ARS_PROFILE_ATTR_PORT_LOAD_FUTURE are enabled.
      *
-     * @type sai_uint8_t
+     * @type sai_u8_list_t
```

Comment:
I have a major concern that having separate bands for past and future might be h/w specific and will discuss that in a separate comment!

Comments from syntax perspective:

1.  non backward compatible change.
2.  It would be better to have a new attribute like SAI\_ARS\_PROFILE\_ATTR\_QUANT\_BANDS\_PAST, and SAI\_ARS\_PROFILE\_ATTR\_QUANT\_BANDS\_FUTURE